web/templates/navbar.hbs

```diff
@@ -46,11 +46,16 @@
                 </div>
             </div>
             <div id="userlist-toggle" class="hidden-for-spectators tippy-zulip-delayed-tooltip" data-tooltip-template-id="userlist-tooltip-template">
-                <a id="userlist-toggle-button" role="button" class="header-button" tabindex="0"><i class="fa fa-group" aria-hidden="true"></i>
+                <a id="userlist-toggle-button" role="button" class="header-button" tabindex="0">
+                    <i class="zulip-icon zulip-icon-triple-users" aria-hidden="true"></i>
```

Comment:
This use of `aria-hidden=true` is unnecessary, I believe, here and throughout. To the best of my knowledge, empty elements like this `<i></i>` aren't rendered to screenreaders anyway.

Reply:
That seems like should be a separate change if we want to do.

Reply:
Yeah. But I think it's worth not including in it new lines, or lines that are being changed anyway.

Reply:
Okay, I removed them. This needs #frontend discussion though. I will just let you start one for this. 🙈